server/index.js

```diff
@@ -52,6 +51,12 @@ router.get('/gamereviews/:gameid', async (req, res) => {
       helpful = data;
       recent = data.slice().sort((a, b) => Date.parse(a.date_posted) > Date.parse(b.date_posted) ? -1 : 1);
     }
+    // console.log({
```

Comment:
No criticism here, but I just wanted to comment that putting objects inside console.logs is an awesome trick I just learned myself not too long ago!